tests/Integration/WinkPostTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Wink\Tests\Integration;
+
+use Wink\WinkPage;
+use Wink\WinkPost;
+use Wink\Tests\TestCase;
+
+class WinkPostTest extends TestCase
+{
+    /** @test */
+    public function a_guest_may_not_hit_any_post_endpoint()
```

Comment:
I think the Laravel naming convention would be `test_a_guest_may_not_hit_any_post_endpoint()` without the `/** @test */` docblock